core/src/main/java/forklift/source/decorators/Queues.java

```diff
@@ -1,14 +1,17 @@
-package forklift.decorators;
+package forklift.source.decorators;
```

Comment:
Moving these files will force a refactor on those upgrading.  Might be ok but wanted to note it.